responses/de/HassGetState.yaml

```diff
@@ -12,6 +12,17 @@ responses:
           Nein
         {% endif %}
 
+      einzeln_zone: |
```

Comment:
i think this is to complex and may lead to confusing replies like:
question: Ist Jonas auf Arbeit?
answer: Nein, ist nicht zu Hause

so we should just reply with ja/nein. if the user want's to know, where a person is, the user can ask for it ^^